records_mover/db/vertica/loader.py

```diff
@@ -66,6 +69,7 @@ def load_from_fileobj(self,
             if rawconn is not None:
                 rawconn.close()
 
+    # TODO: Do I really need this?  Or could I declare records directory as none?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker